src/st/no0/e_diplocephalus_2.c

```diff
@@ -0,0 +1,326 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "no0.h"
+
+// Fireball
+INCLUDE_ASM("st/no0/nonmatchings/e_diplocephalus_2", func_us_801D0990);
+
+extern Entity D_8007A958;
+extern u16 D_us_80180B48[];
+extern u8 D_us_80181DBC[];
+extern u8 D_us_80181DDC[];
+extern u8 D_us_80181DEC[];
+extern u8 D_us_80181E0C[];
+extern u8 D_us_80181E30[];
+
+// Tail entity
+void func_us_801D0E7C(Entity* self) {
```

Comment:
You can rename this here and e_init to `EntityDiplocephalusTail`
Similarly with the `D_us_80180B48` EInit